config/settings.yml

```diff
@@ -825,11 +825,13 @@
     :password:
     :port:
     :user:
+    :scheme:
```

Comment:
Why is this change being made?

Reply:
It's for socks5 proxy. If leave empty or "http", `VMDB::Util.http_uri_proxy` will return a http proxy uri. If scheme is "socks5", it will construct a `socks5://...` proxy. Both http proxy and socks5 proxy works when putting in `http_proxy` env var.
VMDB::Util.http_uri_proxy: https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/util.rb#L3